src/main/javascript/view/MainView.js

```diff
@@ -276,6 +277,6 @@ SwaggerUi.Views.MainView = Backbone.View.extend({
       var t = n.parents('.endpoint').first().offset().top;
       r.removeClass('is-collapsing');
       o(t);
-    }, 500);
+    }, 150);
```

Comment:
I think that this is one of the time values that controls the animated behavior of expanding/collapsing the right sidebar. In this PR I made the animation much faster but I didn't make it zero. It just felt odd to make the entire page layout change instantly. 

Is there any reason not to try the timings in this PR and then revisit the numbers with Jesse and Matt later?